lib/xcpretty/parser.rb

```diff
@@ -107,7 +107,7 @@ module Matchers
     # $2 = test_suite
     # $3 = test_case
     # $4 = reason
-    FAILING_TEST_MATCHER = /^\s*(.+:\d+):\serror:\s[\+\-]\[(.*)\s(.*)\]\s:(?:\s'.*'\s\[FAILED\],)?\s(.*)/.freeze
+    FAILING_TEST_MATCHER = /^\s*(.+?:\d+):\serror:\s[\+\-]\[([^\s]*)\s([^\]]*)\]\s:(?:\s'.*'\s\[FAILED\],)?\s(.*)/.freeze
```

Comment:
`[^:]+` is still better than `.+?` because you know exactly when to stop.

Reply:
I did this already, but the test failed, so I gave up.
Gave it another shot and I think there's a problem in the test itself (it has a newline). Fixes this.